traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -142,7 +143,21 @@ func (dss *TODeliveryServiceServer) readDSS(tx *sqlx.Tx, user *auth.CurrentUser,
 		orderby = "deliveryService"
 	}
 
-	query, err := selectQuery(orderby, strconv.Itoa(limit), strconv.Itoa(offset))
+	tenancyEnabled, err := tenant.IsTenancyEnabledTx(tx.Tx)
```

Comment:
late to the party, but tenancy is always enabled by default now so this check is unnecessary

Reply:
But can it be turned off?

Reply:
I think we decided to do that, but I don't think we actually did. The code still checks `use_tenancy` everywhere, and I think it's considered false if it doesn't exist. 

I think this because the API Tests until recently broke horribly if you created that Parameter, suggesting Tenancy is disabled by default.

Also, I'm not opposed to making Tenancy "always on," but I do kind of think we should enable Capabilities at the same time (https://github.com/apache/trafficcontrol/pull/2791). Tenancy isn't really useful without Capabilities to limit what users can do, and it could be deceptive, a CDN operator might think tenancy limits an admin/ops user, when it doesn't.

Reply:
This has been my understanding, that tenancy is enabled by default, but still optional - though I have heard that the decision to make it not optional has been made (and not actually done yet).

Reply:
It can be "turned off" by setting the parameter to false or deleting the parameter, but it is not really "turned off" at that point because we already have various places in the code that assume tenancy is always enabled (so they don't check the parameter). So "turning it off" doesn't completely turn it off, which is why we should really be preventing the deletion/update of the `use_tenancy` parameter to "turn off tenancy", since that really just leaves the system in an inconsistent state. Some things will check the tenancy parameter, whereas others already assume tenancy is always enabled.

We attempted to delete the `use_tenancy` parameter from the system but realized that there is too much existing code that depends on that parameter being there. The plan was to "fix" the existing code that checks the parameter by removing the parameter check and just assume that tenancy is always enabled. This was all discussed on the mailing list already; the decision to have it always enabled has already been made. Steps have already been made to enable it by default, but the existing code that checks the `use_tenancy` parameter has not been cleaned up yet.

tldr; tenancy should always be considered enabled, and existing code that checks the `use_tenancy` parameter needs to be fixed. Once all existing tenancy checks have been "fixed", the parameter can be deleted.

Reply:
I definitely agree that should all be done, and the sooner the better. But until the `use_tenancy` parameter can actually be removed so that tenancy is a hard requirement of ATC it just seems safer to keep checking.

Reply:
> But until the `use_tenancy` parameter can actually be removed so that tenancy is a hard requirement of ATC it just seems safer to keep checking.

Then we should do our best to remove all the usages of the `use_tenancy` parameter from the codebase. I think the usage is fairly self-contained within helper functions, so maybe we can just change the helper functions to just return `true`. Then we can remove all the usages of the now-useless helper functions to clean up the code.